examples/c/profile.c

```diff
@@ -220,7 +229,13 @@ int main(int argc, char *const argv[])
 		/* Set up performance monitoring on a CPU/Core */
 		pefd = perf_event_open(&attr, pid, cpu, -1, PERF_FLAG_FD_CLOEXEC);
 		if (pefd < 0) {
-			fprintf(stderr, "Fail to set up performance monitor on a CPU/Core\n");
+			if (errno == ENOENT) {
```

Comment:
Probably confusing if we fail to set up the software event but recommend that as a work around:
```suggestion
			if (errno == ENOENT && !sw_event) {
```

Reply:
You are right, only need to print hint without `--sw-event`.